.gitignore

```diff
@@ -8,6 +8,6 @@
 /spec/reports/
 /tmp/
 /*.gem
-
+.idea
```

Comment:
Hmmm, putting user specific ignores in the .gitignore doesn't strike me as a best practice. I went and had a look at how Rails handles this, and they don't do it, but advise putting .idea in your global gitignore (makes sense!)

https://github.com/rails/rails/blob/master/.gitignore#L1

Reply:
@bessey  :+1: